docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -101,6 +145,27 @@ ALTER TABLE table_identifier { ALTER | CHANGE } [ COLUMN ] col_spec alterColumnA
 
     Change column's definition.
 
+### REPLACE COLUMNS
+
+`ALTER TABLE REPLACE COLUMNS` statement removes all existing columns and adds the new set of columns.
+Note that this statement is only supported with v2 tables.
+
+
+#### Syntax
+
+```sql
+ALTER TABLE table_identifier REPLACE COLUMNS  
+  [ ( col_name1 col_type1 [ COMMENT col_comment1 ], ... ) ]
+```
```

Comment:
This one is a little complicated. Here is the syntax:
```
    | ALTER TABLE table=multipartIdentifier partitionSpec?
        REPLACE COLUMNS
        '(' columns=qualifiedColTypeWithPositionList ')'    

qualifiedColTypeWithPositionList
    : qualifiedColTypeWithPosition (',' qualifiedColTypeWithPosition)*
    ;

qualifiedColTypeWithPosition
    : name=multipartIdentifier dataType (NOT NULL)? commentSpec? colPosition?
    ;
```
Could you please make the syntax more accurate?

Reply:
thanks for the review. fixed.